### PR TITLE
Do not validate certificate when downloading zeromq

### PR DESCRIPTION
**Pull request by stuartwan**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@
   get_url:
     url: "https://archive.org/download/zeromq_{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz"
     dest: "/tmp/zeromq-{{ zeromq_version }}.tar.gz"
+  validate_certs: no
```
Comment (tjoelsson) [inline tasks/main.yml]: `validate_certs` is an option of `get_url`. It should be indented 2 more spaces.

Comment (stuartwan) [inline tasks/main.yml]: Yes my vim tricked me, already updated.

**Comment by tjoelsson** [comment]
👍

**Comment by wanleung** [comment]
👍

**Comment by HouCoder** [comment]
👍 
I had this problem yesterday, I fixed it by manually install ZeroMQ in my box.:sob:

**Pull request merged**